metrics/dune-dashboard.md

```diff
@@ -0,0 +1,190 @@
+# Pyth Dune Dashboard
+
+Find the full Dune Dashboard [here](https://dune.com/cctdaniel/pyth-oracle).
+
+---
+
+This dashboard was created by a Pyth contributor to further provide relevant statistics around the Pyth network, its operations and price feed, as well as its usage by protocols and users.
+
+If you have any relevant metrics you’d like to see on this dashboard, come let us know in our [Discord](https://discord.gg/invite/PythNetwork).
+
+---
+
+### Price Accounts
+
+All price feed IDs can be found [here](https://pyth.network/developers/price-feed-ids/).
+
+**Past 1W Price Accounts Count — Pyth Oracle Price Accounts Count (1W)**
+
+![Past 1W Price Accounts Count](../.gitbook/assets/dune/past_1w_price_accounts_count.png)
+
+This data point tracks how many Pyth price feeds have received at least one publisher price update in the past week. 
+
+**Past 1W Price Updates — Pyth Oracle Price Accounts (1W)**
+
+![Past 1W Price Updates](../.gitbook/assets/dune/past_1w_price_updates.png)
+
+This table returns all Pyth price accounts that received at least one publisher price update in the past week, the number of active publishers that contributed to it, as well as the total count of price updates provided by all publishers.
+
+*The Crypto.USDT/USD price feed saw 14 different publishers submitting at least one price update in the past week. In total, these 14 publishers submitted 8.3M distinct price updates on the USDT/USD price feed.*
```

Comment:
Apparently Dune's embed visualizations [doesn't work with gitbook](https://docs.dune.com/sharing/embeds#known-issues), this is more of an example of how to interpret the charts so we don't necessarily have to update it every week, maybe we would have to make it clearer that this is an example so that people don't get confused?